app/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 android {
-    compileSdk 32
```

Comment:
Why is this?

Reply:
That's likely because my phone is old and I was trying to get some things to line up with the debugger. I guess it should be changed back.